guest/filter.go

```diff
@@ -17,6 +17,10 @@
 package guest
 
 import (
+	// override the default GC with a more performant one.
+	// Note: this requires tinygo flags: -gc=custom -tags=custommalloc
```

Comment:
I think this requires all users to add these flags, all to save one line of code. Though one could argue the default is slow enough that build failure by default is reasonable :). If not, is this better as docs/example than in the SDK?

Reply:
long answer short is that I don't expect people will be able to easily figure out how to reduce lack of performance. It is hard enough when you are good at wasm. @kerthcet and @sanposhiho otoh are more authoritative than me on this one (hence approval karma ;))

Anyway, I feel failing the build is probably a feature vs a bug, and would be easy to answer, even if it is cryptic.
```
error: Linking globals named 'runtime.alloc': symbol multiply defined!
make: *** [examples/filter-simple/main.wasm] Error 1
```

Reply:
Cool - I realized, I believe it could be possible to check the tag and panic with an error message in `init` when it's not defined. Will try it out!

Reply:
Yep, we can have a doc for the SDK about how to build the plugin and then it's OK to require people to do that.